Fix PRNG seeds for benches & tests

saeyoon17 opened the pull request:
Fix #15 
I modified some of the codes in the which uses seeding using **from_entropy()** to use **seed_from_u64(0)**.

I modified:
- codes in benches/
- test codes that use from_entropy()

There are still three parts of code that use **from_entropy()**.
- ot/mpz-ot-core/src/chou_orlandi/receiver.rs
- ot/mpz-ot-core/src/chou_orlandi/sender.rs
- share-conversion/mpz-share-conversion/src/sender.rs

Yet I left them since I was not sure whether having fixed seed for them was appropriate.

saeyoon17 committed:
Fix PRNG seeds for benches & tests

themighty1 commented:
thanks, @saeyoon17

themighty1 commented:
Our goal was to use `ChaCha12Rng::seed_from_u64(0)` only in tests in order to make tests reproducible.

saeyoon17 commented:
@themighty1  I see, would it be better if I take back the changes in the benches folder and only leave the change in test case?

themighty1 commented:
@saeyoon17 , sry, I misspoke, we use it both in tests and in benches.

th4s reviewed (changes requested):
Thanks! While you are at it, can you please also adapt `share-conversion/mpz-share-conversion-core/benches/inverse_gf2_128.rs`

>There are still three parts of code that use from_entropy().

    ot/mpz-ot-core/src/chou_orlandi/receiver.rs
    ot/mpz-ot-core/src/chou_orlandi/sender.rs
    share-conversion/mpz-share-conversion/src/sender.rs

Indeed these places should definitely **not** use a fixed seed, since they are not tests.

saeyoon17 committed:
remove entropy & 0u64 to 0

saeyoon17 commented:
Oh, sorry for missing that one. I changed it. 
Also, I modified
`let mut rng = ChaCha12Rng::seed_from_u64(0u64);` 
to 
`let mut rng = ChaCha12Rng::seed_from_u64(0);`
in [gf2_128.rs](https://github.com/privacy-scaling-explorations/mpz/pull/66/commits/cabde7a00dbb65f5d89f03a7dcc0230c64d3f140#diff-91f9a5c1c4430fa407e6e5fcafad4df907c63dc034333b7ce129db6d33f5f7f1) to keep the convention.

th4s reviewed (approved):
thanks!
